cloudfront/wellcomelibrary.org/terraform/dns.tf

```diff
@@ -1,9 +1,45 @@
-resource "aws_route53_record" "prod" {
+resource "aws_route53_record" "prod-internal" {
   zone_id = data.aws_route53_zone.zone.id
   name    = "wellcomelibrary.org"
   type    = "A"
   records = ["195.143.129.236"]
-  ttl     = "300"
+  ttl     = "60"
+
+  weighted_routing_policy {
+    weight = 0
+  }
+
+  set_identifier = "internal"
+
+  provider = aws.dns
+}
+
+resource "aws_route53_record" "prod-cloudfront" {
+  zone_id = data.aws_route53_zone.zone.id
+  name    = "wellcomelibrary.org"
+  type    = "A"
+
+  weighted_routing_policy {
+    weight = 50
+  }
+
+  alias {
+    name                   = "d2cksn8op995w4.cloudfront.net"
```

Comment:
Where does this CloudFront name come from?

Reply:
Same issue as with zone_id.